tbx/core/blocks.py

```diff
@@ -329,6 +329,28 @@ class Meta:
         template = "patterns/molecules/streamfield/blocks/work_chooser_block.html"
 
 
+class PromoBlock(blocks.StructBlock):
+    title = blocks.TextBlock()
+    description = blocks.TextBlock()
+    image = ImageChooserBlock()
+    button_text = blocks.CharBlock(max_length=55)
+    button_link = blocks.StreamBlock(
+        [
+            ("internal_link", blocks.PageChooserBlock()),
+            ("external_link", blocks.URLBlock()),
+            ("email", blocks.EmailBlock()),
+        ],
+        required=True,
+        max_num=1,
+    )
+    secondary_link = LinkBlock(required=False)
```

Comment:
Nitpick: this is appearing in the editor just as 'Link' - could it say 'Secondary link' to be clearer for editors?
<img width="986" alt="Screenshot 2024-02-26 at 16 01 14" src="https://github.com/torchbox/torchbox.com/assets/771869/ce4e3096-eca3-48ac-909c-7a898b2b8492">